### PR TITLE
[OCPNODE-883] promote e2e-aws-crun to required

### PR DESCRIPTION
**Pull request by rphillips**
Promote e2e-aws-crun to required.

ref: https://issues.redhat.com/browse/OCPNODE-883

### CODE REVIEW

File: ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml [hunk 1]
```diff
@@ -404,7 +404,6 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-cgroupsv2
 - as: e2e-aws-crun
-  optional: true
```
Comment (harche) [inline ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml:407]: Should we explicitly mark it required by setting `optional: false`? Maybe this we don't rely on default value to be always `true`. If for some reason they decide to change the default behaviour of that `optional` field, it won't affect this job if we have it `optional: false` explicitly set. WDYT?

Comment (rphillips) [reply ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml:407]: good point... fixed

Comment (rphillips) [reply ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml:407]: I'll have to revert... the change fails on ci-operator-config-metadata's lint.

**Comment by rphillips** [comment]
[Test Grid](https://testgrid.k8s.io/redhat-openshift-ocp-release-4.11-informing#periodic-ci-openshift-release-master-ci-4.11-e2e-aws-crun) is passing except for occasional VPC limit errors.

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**